qcodes/math/field_vector.py

```diff
@@ -191,12 +197,12 @@ def set_component(self, **new_values):
             # kept constant and only r is changed. After changing r,
             # (x, y, z) values are recomputed, as is the rho coordinate.
             # Internally we arrange this by setting x, y, z and rho to None
-            # and calling self._compute_unknowns()
+            # and calling self._compute_unknowns().
             >>> f.set_component(r=10)
 
         Args:
             new_values (dict): keys representing parameter names and values the
-                values to be set
+                               values to be set.
```

Comment:
A question that probably reveals my ignorance about sphinx: why is this extra indentation needed?

Reply:
As far as I know it's not https://www.sphinx-doc.org/en/1.5/ext/example_google.html

Reply:
I see. Should we have a standard for indentation? We should probably follow `Sections are created with a section header and a colon followed by a block of indented text.` from that link.

Reply:
Or rather:
```
 The format for a parameter is::

        name (type): description
            The description may span multiple lines. Following
            lines should be indented. The "(type)" is optional.

            Multiple paragraphs are supported in parameter
            descriptions.
```

Reply:
I guess what I'm trying to say is that
```
            new_values (dict): keys representing parameter names and values the
                values to be set
```
is to be preferred over
```
            new_values (dict): keys representing parameter names and values the
                               values to be set.
 ```

Do you agree? Do we care?

Reply:
In this case I believe the default is and should be indent by 4 spaces

Reply:
> In this case I believe the default is and should be indent by 4 spaces

+1